### PR TITLE
Add Travis Build

### LINKED ISSUE #156: Create addons-robot npm account and deploy new tags from Travis
Make sure the `dist/` folder is created and the `tests/` folder isn't published to npm, as we just want the usable binary for that.

### PR DESCRIPTION
**Pull request by tofumatt**
Account is created (https://www.npmjs.com/~addons-robot), so this fixes #156.

### CODE REVIEW

**Comment by tofumatt** [comment]
Okay, all fixed up. Another quick r? whenever

File: README.md [hunk 1]
```diff
@@ -210,3 +211,10 @@ same message could be an error *or* a warning for example.
 
 Lastly when the processing is complete the validator will output
 the collected data as text or JSON.
+
+## Deploys
+
+We deploy to npm automatically using TravisCI. To release a new version,
+increment the version in `package.json` and create a PR. After the PR is
+merged create a new release with the same tag name as your new version.
```
Comment (muffinresearch) [inline README.md]: Could you add a link to the releases page on the project?

You could also make a note that the release title and tag should be a version e.g. `0.0.7`

**Comment by muffinresearch** [comment]
Private is still set to true.

r+wc

File: .travis.yml [hunk 2]
```diff
@@ -11,3 +11,13 @@ notifications:
...
+deploy:
+  provider: npm
+  email: addons-dev-automation+npm@mozilla.com
+  before_deploy: npm run build
+  api_key:
+    secure: bVMeDrmxJB7QajecwgqoKlIfdduwUv+qqGrv1ucD9IS9aD2fFUljMruzVxzYzJLgdz1anWHEeuLsVZ4KbCgp8axa/PHKjVOhqglRXvgJezE+d4DuVtbBOXAD4JV+F8DA75ZLCe+sZm94lFa4MKnUiyvwR/v2yyNMQWJL/pZRv87yx/G1i3yBuMzhFaj51DQSWreXyxq+mI5TLvLoWO9jzXtFOCynq+ufFhWw7pUz3/eqwp1VD9YIuUxLsT9U0KJHF4E5flhWl3tq9OYM3k26qs0m7BUYZ5O1dSnXQsODlbSe1PnRs6o1Mex/ZYdN56CdP4+32o1seGWbWHQRzvQuHSR29opqPORuKcyDAbXeUhyoo1V6zguKbXb7++9iSq0CiKLSHMzeuTco7bdpeOZYnWSH1mBi/QMUIg2dp7+0fpRTuBp7+rY2xFzqoRGg16fLotVkZDBwlXkxRTatM0QVvvscAmksPBIZI8xmB2+B0RNLcwBTuQKVeQ7988Bw1oZ94rhIhfKljbhnIzU11Y1sexmV/UzXxiFs/L6wbFr0fI7BTBrfdXAIMWaGQ9uwBCRn2EH9fLx3B3LoEXe7eL84jLgdOgRzR6eQTlkDpwvh9Jf3fbcv1S89pWpwt2CgxtofY/wMO3WpRW6e5fqo83rKwmuAzDuqzRjl+vvoc4IR5yE=
+  on:
+    tags: true
+    repo: mozilla/addons-validator
+    branch: travis-build
```
Comment (tofumatt) [inline .travis.yml]: Just caught this, should be master!

File: README.md [hunk 3]
```diff
@@ -210,3 +211,10 @@ same message could be an error *or* a warning for example.
...
+Once the build passes it will deploy. Magic! ✨
```
Comment (muffinresearch) [inline README.md]: magic

**Pull request merged**